src/shared/cache/SampleListsInStudyCache.ts

```diff
@@ -0,0 +1,39 @@
+import LazyMobXCache, {AugmentedData} from "../lib/LazyMobXCache";
```

Comment:
@adamabeshouse we should be using new cache, yes?

Reply:
yeah, definitely

Reply:
@alisman @adamabeshouse new cache? example?

Reply:
you mean `MobxPromiseCache`

Reply:
@kalletlak yes, let me know if you have questions about how to use it. I think you'll find its cleaner/easier